assets/files/etc/NetworkManager/dispatcher.d/brextscript

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
from NetworkManager man page I learn that this script is executed quite often. I don't think it should ever generate another ifcfg file or modify its own con state. Wouldn't it create an endless mess?

        NetworkManager will execute scripts in the /etc/NetworkManager/dispatcher.d directory or subdirectories in alphabetical order in
       response to network events. Each script should be a regular executable file owned by root. Furthermore, it must not be writable by
       group or other, and not setuid.

       Each script receives two arguments, the first being the interface name of the device an operation just happened on, and second the
       action. For device actions, the interface is the name of the kernel interface suitable for IP configuration. Thus it is either
       VPN_IP_IFACE, DEVICE_IP_IFACE, or DEVICE_IFACE, as applicable. For the hostname and connectivity-change actions it is always "none".

Reply:
changed so it will only generate needed file once